vipe/graphviz/converter.py

```diff
@@ -34,37 +35,37 @@ def __init__(self, detail_level, show_input_ports, show_output_ports,
         """
         score_map = ImportanceScoreMap(detail_level)
         self.__b = DotBuilderWrapper(
-                            score_map, show_input_ports, show_output_ports,
-                            vertical_orientation)
+            score_map, show_input_ports, show_output_ports,
+            vertical_orientation)
         self.__low_score_nodes_remover = LowScoreNodesRemover(score_map)
         self.__input_created = False
         self.__output_created = False
         self.__already_run = False
-    
+
     def run(self, pipeline):
         """
         Convert Pipeline to its visualization in GraphViz dot format
-        
+
         Args:
             pipeline (Pipeline): input Pipeline.
-        
+
         Return:
             string: dot format
         """
-        assert self.__already_run == False
+        assert self.__already_run is False
```

Comment:
Isn't it possible to simply assert
```assert self.__already_run```
without explicitly comparing to boolean value? Using ```not``` when comparing to false.